client/app/cdn/dedicated/manage/ssl/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We're using the same texts in the Web manager, isn't there a way to mutualize the texts ?